src/component.ts

```diff
@@ -7,11 +7,12 @@ import {
   Component,
   ContextTags,
   injectable,
-  ServiceOrProviderClass,
+  ServiceOrProviderClass
```

Comment:
Please run `npm run prettier:fix` to format the code.

Reply:
Done. Thanks for the notice!